data.sql

```diff
@@ -0,0 +1,6 @@
+
+
+INSERT INTO animals (name, date_of_birth, escape_attempt, neutered, weight_kg) VALUES ('Aguman', '2020-02-03', 0, true, 10.23);
```

Comment:
- [ ] The name of the animal is `Agumon` and not `Aguman`. The implication of inserting incorrect data into the database is that queries will not return the correct results like the first query in your `queries.sql` file. Kindly insert the correct name. 💻 😎 

When you make the change, kindly update the pull request description with the screenshots of the correct result. 👍